rdflib/plugins/memory.py

```diff
@@ -249,6 +262,7 @@ def namespaces(self):
             yield prefix, namespace
 
     def add(self, triple, context, quoted=False):
+        count_before = self.__len__()
```

Comment:
Why not use `len(self)`?

Reply:
That should work too. I thought `self.__len__()` would be a direct call and `len(self)` would be an indirect call, thus I preferred the former. Should I change it to `len(self)`?